taildrop/send.go

```diff
@@ -167,7 +158,9 @@ func (m *Manager) PutFile(id ClientID, baseName string, r io.Reader, offset, len
 	}
 
 	// Copy the contents of the file.
-	copyLength, err := io.Copy(inFile, r)
+	h := sha256.New()
+	w := io.MultiWriter(inFile, h)
```

Comment:
I think we just had this for debugging, so I think you can get rid of the MultiWriter and just copy directly to `inFile` again.

Reply:
yeah i def missed some stuff when i was stripping out our debugging.